radix-engine/src/system/transaction/multithread_txn_processor.rs

```diff
@@ -0,0 +1,138 @@
+use crate::blueprints::transaction_processor::{
+    ResumeResult, TxnProcessorThread, MAX_TOTAL_BLOB_SIZE_PER_INVOCATION,
+};
+use crate::errors::{KernelError, RuntimeError};
+use crate::kernel::kernel_callback_api::KernelCallbackObject;
+use crate::system::actor::{Actor, FunctionActor};
+use crate::system::system::SystemService;
+use crate::system::system_callback::{System, SystemBasedKernelApi};
+use crate::system::system_modules::auth::AuthModule;
+use radix_common::constants::TRANSACTION_PROCESSOR_PACKAGE;
+use radix_common::prelude::{BlueprintId, GlobalAddressReservation};
+use radix_engine_interface::blueprints::transaction_processor::{
+    TRANSACTION_PROCESSOR_BLUEPRINT, TRANSACTION_PROCESSOR_RUN_IDENT,
+};
+use radix_rust::prelude::*;
```

Comment:
Import wise, lots of this can be replaced with `radix_engine::internal_prelude::*` to avoid lots of import churn if we move things.